### PR TITLE
user ssl files need to be present

### PR DESCRIPTION
**Pull request by koaps**
user ssl files need to be present or they won't be updated on cert changes, ran into this testing in docker.

### CODE REVIEW

**Comment by igalic** [comment]
@koaps a bit late to ask, but would you mind adding tests for this?

File: manifests/user.pp [hunk 1]
```diff
@@ -39,6 +39,7 @@
 
   if $middleware_ssl or $securityprovider == 'ssl' {
     file { "${homedir}/.mcollective.d/credentials/certs/ca.pem":
+      ensure => 'present',
```
Comment (petems) [inline manifests/user.pp]: `ensure => present` is ambiguous, it'll return true for simlinks and directorys. It's better to do `ensure=>file` :+1: 

See https://github.com/puppet-community/puppet-lint-file_ensure-check

**Comment by jyaworski** [comment]
Can you rebase?

**Comment by petems** [comment]
Closed in favor of #307

**Pull request closed without merging**